website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,543 @@
+   with the JSON data:
+
+    ```shell-session
+    $ curl                                             \
+      --request POST                                   \
+      --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+      --data @data.json                                \
+      ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/<unique_rule_name>
+    ```
+
+    For example, to create a new login rule called "dev-gui-login":
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ curl                                               \
+        --request POST                                   \
+        --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+        --data @data.json                                \
+        ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+    ```
+
+    </CodeBlockConfig>
+
+  `/sys/config/ui/login/default-auth/{rule_name}` does not return data on success.
+
+1. Make a `GET` call to
+   [`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+   with the new rule name to verify the login configuration.
+
+   For example:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ curl                                               \
+        --request GET                                    \
+        --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+        ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+    ```
+
+    </CodeBlockConfig>
+    <br/>
+    <CodeBlockConfig hideClipboard>
+
+    ```json
+    {
+      "data": {
+        "backup_auth_types": [
+          "ldap",
+          "token"
+        ],
+        "default_auth_type": "userpass",
+        "disable_inheritance": false,
+        "namespace_path": "dev/"
+      },
+    }
+    ```
+
+    </CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+
+## Delete a login rule
+
+<Tabs>
+<Tab heading="CLI" group="cli">
+
+Use `vault delete` with the 
+[`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+endpoint and rule name to delete a login setting for a namespace. 
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/<unique_rule_name>
+```
+
+For example:
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/dev-gui-login
+```
+
+</CodeBlockConfig>
+
+</Tab>
+<Tab heading="GUI" group="gui">
+
+You can delete a login rule in two places from the **UI Login Settings** page.
+
+**From the additional options menu for the customization**:
+
+1. Find the login rule you want to delete.
+1. Click the additional option button (three dots).
+1. Select **Delete** from the dropdown menu to bring up the delete confirmation modal.
+
+**From the customization details page**:
+
+1. Find the login rule you want to delete.
+1. Select it to open the login settings details page.
```

Comment:
The use of "it" in the second statement had me eyeing this. Whenever possible, especially for instructions, I try to be explicit and instead of using pronouns like "it" which refers to "login rule" I'll just use "login rule" so it's super clear what I'm referring to.
```suggestion
1. Select the login rule to open its login settings details page.
```